src/planner.cc

```diff
@@ -663,175 +668,205 @@ void Planner::Impl::bring_path_to_halt() {
   issue_motor_move_if_possible(true);
 }
 
+// Compute the backard pass of <segment_to_plan> given the
+// information (such as starting speed and defining axis) of the <next_segment>.
+// If <next_segment> is nullptr, the next segment is assumed to not exist (end
+// of the buffer). The function returns true if the final speed already connects
+// with the starting speed of the next segment and this is not the last segment.
```

Comment:
The return value might be slightly confusing. The function is named `BackwardUpdateSegment()` so one would expect that `true` actually means that it was updated.
Might be worthwhile reconsidering the return value.

Reply:
(solved)